packages/node/src/rpc/index.ts

```diff
@@ -11,6 +15,9 @@ import type {
 	UnsubscribeDRPRequest,
 	UnsubscribeDRPResponse,
 } from "../proto/drp/node/v1/rpc_pb.js";
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+const PROTO_PATH = path.resolve(__dirname, "../proto/topology/node/rpc.proto");
```

Comment:
```suggestion
const protoPath = path.resolve(dirname(fileURLToPath(import.meta.url)), "../proto/topology/node/rpc.proto");
```

move this inside the init function